lib/demos/stack_demo_view.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_full_learn/core/random_image.dart';
+
+class StackDemoView extends StatelessWidget {
+  const StackDemoView({super.key});
+  final _cardHeight = 50.0;
```

Comment:
Add space between class constructor and variables.

```suggestion
  const StackDemoView({super.key});
  
  final _cardHeight = 50.0;
```

Reply:
Done.